content/docs/rfcs/51/README.md

```diff
@@ -66,11 +66,8 @@ shard clusters are divided into ranges:
 
 | index (range) |    usage                 |
 |    ---        |    ---                   |
-|             0 |   global                 |
```

Comment:
simplying this:
* 49152 - 65535 | automatic sharding -> no longer true
* what does "specific app protocols" and "all app protocols" even mean